File: app/src/main/java/com/xuie/imaginaryandroid/gui/show/Level0Item.java
package com.xuie.imaginaryandroid.gui.show;

import com.chad.library.adapter.base.entity.AbstractExpandableItem;
import com.chad.library.adapter.base.entity.MultiItemEntity;

/**
 * Created by xuie on 17-8-15.
 */

public class Level0Item extends AbstractExpandableItem<Level1Item> implements MultiItemEntity {
    private String type;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    @Override
    public int getLevel() {
        return 0;
    }

    @Override
    public int getItemType() {
        return ExpandableItemAdapter.TYPE_LEVEL_0;
    }
}

File: app-mvp-kotlin/src/main/java/xuk/imaginary/gui/gank/show/Level0Item.kt
package xuk.imaginary.gui.gank.show

import com.chad.library.adapter.base.entity.AbstractExpandableItem
import com.chad.library.adapter.base.entity.MultiItemEntity

/**
 * Created by xuie on 17-8-15.
 */

class Level0Item : AbstractExpandableItem<Level1Item>(), MultiItemEntity {
    var type: String? = null

    override fun getLevel(): Int {
        return 0
    }

    override fun getItemType(): Int {
        return ExpandableItemAdapter.TYPE_LEVEL_0
    }
}
